script/Ciel/Day 13/13_01_01_CIEL13_2.txt

```diff
@@ -387,17 +411,19 @@ Even though it's already dead, this lump of flesh is pretending to be alive, and
 [sha:b5c5c04f329ab00a006689e35a5574dac03dd509]{
 -- Page 94, Offset 33420.
 -- 自分でも聞き取れない叫び声を上げて走る。
-I let out a scream that even I can't comprehend before running away.
+Deaf to my own scream, I break into a run.
 }
 [sha:1b1cbc3ce605c8419fc27c1ba5a4828ed784c520]{
 -- Page 94, Offset 33421.
 -- 今までまったく気がつかなかった景色の中を走る。
-I continue running in the midst of the scenery that I completely failed to take in until moments ago.
+I run through a reality I'd been blind to up until now.
 }
 [sha:cf0b86ba43604431d21a9848727c296a9cfa8c97]{
 -- Page 94, Offset 33422.
 -- 廊下には隠しようもないほどの血の匂い。血の匂い。
-The conspicous smell of blood that's drifting throughout the corridor. This bloody stench.
+// Duplicated 血の匂い sounds kinda dumb in EN imo
+The hallway is soaked with the unmistakable stench of blood.
+ A dull, iron, tang.
```

Comment:
Yeah, a little bit. I feel like the intention is to make it feel like its everywhere he goes? It's inescapable, essentially. Maybe if there's a qualifier added like that It'll be easier to swallow the duplication. Either that or some other phrasing. Either way I think the current rendition isn't ideal.

>The hallway is soaked with the unmistakable stench of blood.
>Everywhere I go, the stench of blood.